pkg/usagestats/config.go

```diff
@@ -0,0 +1,13 @@
+package usagestats
```

Comment:
have we pursued finding a shared location for this code instead of forking it?

@mattmendick

Reply:
There was a conversation in a tech leads channel about creating a place for some of this, since folks are noticing the duplication.  There is room to split this up some and move pieces to a shared location, but some items seem somewhat specific to tempo.  The items that I'm thinking of are the storage clients to the various backends and their associated configurations as well as some error handling.

A `backend-libs` repo could exist and start moving components.  Perhaps some coordination makes sense.

Reply:
I think forking it is a good short term solution this minute, but I would really like to see a shared library for this kind of thing. If there is time to create this and start contributing things to it, I think that would be a win.

Reply:
I'll leave it to y'all to discuss if there is time for that added overhead, but it would be a net positive on the databases group to take the time to do this.

Reply:
There doesn't seem to be much appetite from the leads for sharing this code among the backend databases, at least for now.  We could revisit at a later time, but not sure when we would.